app/models/audit_results.rb

```diff
@@ -148,33 +126,12 @@ def to_json(*_args)
 
   private
 
+  attr_reader :result_array
```

Comment:
Appreciate being intentional about what's in the public API and what's an implementation detail.